resources/js/functions.js

```diff
@@ -312,40 +312,17 @@ $(document).ready(function () {
   }
 });
 
-$(document).ready(function () {
-  $('.tabsection').each(function () {
-    var tabsection = this;
-    $(tabsection).find('.nav-tab > .item-tab > .item-tab-link').each(function () {
-      var tabLink = this;
-      var targetTab = $(tabLink).attr('data-target');
-      $(tabLink).click(function () {
-        console.log("clicked on " + targetTab);
-        $(tabsection).find('.nav-tab > .item-tab > .item-tab-link').each(function() {
-          var otherTab = this;
-          var otherTarget = $(otherTab).attr('data-target');
-          otherTarget === targetTab ? $(otherTab).addClass('active') : $(otherTab).removeClass('active');
-        })
-        $(tabsection).children('.tabcontent').each(function() {
-          var tabContent = this;
-          var tabId = $(tabContent).attr('data-tab');
-          targetTab === tabId ? $(tabContent).addClass('active') : $(tabContent).removeClass('active');
-        });
-      });
-    });
-  });
-});
-
 $(document).ready(function () {
   // click the get-started tab corresponding to the users OS.
   if ($(".main-download").length) {
     var os = getOS();
     if (os === 'unix') {
       os = 'linux';
     }
-    $("#install-cs-setup-tabs").find('.nav-tab > .item-tab > .item-tab-link').each(function () {
+    $("#install-cs-setup-tabs").find('input').each(function () {
       var targetTab = $(this).attr("data-target");
       if (targetTab === os) {
-        $(this).click();
+        $(this).prop("checked", true);
       }
     });
```

Comment:
The loop could probably be replaced by more precise selector:

```suggestion
    $("#install-cs-setup-tabs").find('input[data-target=' + os + ']').prop("checked", true);
```